lib/matrix_sdk/protocols/cs.rb

```diff
@@ -767,6 +767,30 @@ def get_room_state_all(room_id, **params)
     request(:get, :client_r0, "/rooms/#{room_id}/state", query: query)
   end
 
+  # Retrieves number of events that happened just before and after the specified event
+  #
+  # @param room_id [MXID,String] The room to get events from.
+  # @param event_id [MXID,String] The event to get context around.
+  # @option limit [Integer] (10) The limit of messages to retrieve
+  # @option params [String] :filter A filter to limit the retrieval to
+  # @return [Response] A response hash with contextual event information
+  # @see https://matrix.org/docs/spec/client_server/r0.6.1#get-matrix-client-r0-rooms-roomid-context-eventid
+  #      The Matrix Spec, for more information about the call and response
+  # @example Find event context with filter and limit specified
+  #   api.get_room_event_context('##room_id##', '##event_id##', filter: { types: ['m.room.message'] }.to_json, limit: 20)
```

Comment:
I'd suggest using things like `#room:example.com` and `$event_id:example.com` or `$exampleMatrixEventId` as the examples, to use the same grammar as what the input is expected to be.